tests/openassetio/test/managerValidator/test_harness.py

```diff
@@ -0,0 +1,222 @@
+        invalid_path = "/i/do/not/exist"
+        with pytest.raises(RuntimeError) as exc:
+            fixturesFromPyFile(invalid_path)
+        assert str(exc.value) == f"Unable to parse '{invalid_path}'"
+
+    def test_when_called_with_non_python_file_then_raises_exception(self):
+        with tempfile_with_contents(".txt", "Hello!") as text_file:
+            with pytest.raises(RuntimeError) as exc:
+                fixturesFromPyFile(text_file.name)
+            assert str(exc.value) == f"Unable to parse '{text_file.name}'"
+
+    def test_when_called_with_python_file_missing_fixtures_var_then_raises_exception(self):
+        with tempfile_with_contents(".py", "cabbages = {}") as malformed:
+            with pytest.raises(RuntimeError) as exc:
+                fixturesFromPyFile(malformed.name)
+            assert str(
+                exc.value) == f"Missing top-level 'fixtures' variable in '{malformed.name}'"
+
+    def test_when_called_with_valid_path_then_returns_expected_fixture_dict(self):
+
+        with tempfile_with_contents(".py", inspect.cleandoc("""
+                from openassetio import constants
+                fixtures = {'ignored': constants.kIgnored}
+                """)) as valid:
+
+            expected_dict = {"ignored": constants.kIgnored}
+            assert fixturesFromPyFile(valid.name) == expected_dict
+
+
+class Test_executeSuite:
+
+    def test_when_called_with_failing_module_then_false_is_returned(
+            self, a_failing_tests_module, executeSuiteTests_fixtures):
+        assert executeSuite(a_failing_tests_module, executeSuiteTests_fixtures) is False
+
+    def test_when_called_with_passing_module_then_true_is_returned(
+            self, a_passing_tests_module, executeSuiteTests_fixtures):
+        assert executeSuite(a_passing_tests_module, executeSuiteTests_fixtures) is True
+
+    def test_when_called_with_executeSuiteTests_module_then_all_tests_pass(
+            self, executeSuiteTests_module, executeSuiteTests_fixtures):
+        assert executeSuite(executeSuiteTests_module, executeSuiteTests_fixtures) is True
+
+    def test_when_called_with_extra_args_then_they_are_passed_to_unittest_main(
+            self, monkeypatch, a_passing_tests_module, executeSuiteTests_fixtures):
+
+        dummyStderr = io.StringIO()
+        monkeypatch.setattr(sys, "stderr", dummyStderr)
+        executeSuite(a_passing_tests_module, executeSuiteTests_fixtures, ["-v"])
+        assert "test_that_will_always_pass" in dummyStderr.getvalue()
+
+
+class Test_FixtureAugmentedTestCase:
+
+    def test_when_constructed_then_objects_are_exposed_via_protected_members(
+            self, a_fixture_dict, mock_session):
+
+        case = FixtureAugmentedTestCase(a_fixture_dict, mock_session)
+        # pylint: disable=protected-access
+        assert case._session == mock_session
+        assert case._fixtures == a_fixture_dict
+        assert case._manager == mock_session.currentManager()
```

Comment:
[consider][minor] 
```suggestion
        assert case._manager == mock_session.currentManager.return_value
```
Otherwise you're adding to the call count of the mock.  Not a problem in this test, just a minor nitpick.

Reply:
Fixed [here](https://github.com/TheFoundryVisionmongers/OpenAssetIO/pull/184/commits/cfed515eebb8263036a24fe1588dcf20a5aa08b7).